test/functional/pdo_sqlsrv/pdo_065_construct_persistent.phpt

```diff
@@ -1,43 +1,46 @@
 --TEST--
 Exception is thrown if the unsupported attribute ATTR_PERSISTENT is put into the connection options
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-include 'MsSetup.inc';
-$dsn = "sqlsrv:Server = $server;database = $databaseName;";
-try{
+require_once("MsSetup.inc");
+try {
     echo "Testing a connection with ATTR_PERSISTENT...\n";
     // setting PDO::ATTR_PERSISTENT in PDO constructor returns an exception
+    $dsn = "sqlsrv:Server = $server;database = $databaseName;";
+    if ($keystore != "none")
```

Comment:
For this particular part perhaps there's no need to enable AE? Just a thought...

Reply:
I was thinking about that too when refactoring the test, but I thought there may be a slight chance that changing the dsn string may affect exceptions being thrown from invalid connection attributes.